.github/workflows/checks_action.yml

```
@@ -0,0 +1,19 @@
+name: "build-test"
+on: [push]
+
+jobs:
+  test_something:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3  # Use the latest version of checkout
+      - name: Create a test output
+        id: test
+        run: echo "::set-output name=summary::Test successful"  # Create a valid output
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)